jbpm-wb-process-runtime/jbpm-wb-process-runtime-client/src/main/java/org/jbpm/workbench/pr/client/editors/instance/diagram/ProcessInstanceDiagramPresenter.java

```diff
@@ -136,6 +139,12 @@ protected void loadProcessInstanceDetails() {
 
             view.setNodeInstances(nodeInstances);
 
+            view.onShowOrHideParentAndSubProcessPanel();
```

Comment:
this should IMO react on an event being fired elsewhere, not to be called ecplicitly.